dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -365,6 +370,8 @@ static void generateDashboard(
       templateData.put("latestArtifacts", latestArtifacts);
       templateData.put("jarLinkageReports", classpathCheckReport.getJarLinkageReports());
       templateData.put("jarToDependencyPaths", jarToDependencyPaths);
+      templateData.put(
+          "jarToSimplifiedPathMessage", jarToDependencyPathSummary(jarToDependencyPaths));
```

Comment:
parameter name no longer matches Java code

Reply:
Updated to `dependencyPathRootCauses`